idea_town/frontend/static-src/styles/modules/_settings.scss

```diff
@@ -68,9 +68,9 @@
 
   a {
     cursor: pointer;
-    text-decoration: none;
     display: block;
     line-height: 2.6rem;
+    text-decoration: none;
```

Comment:
Nothing to see here, just opportunistic chance to fix a-z violation which was causing sass-lint warnings.